frontends/qt/server/Makefile.macosx

```diff
@@ -8,10 +8,11 @@ GOOS=darwin
 darwin:
 	CGO_CFLAGS="-g ${GODARWINSECFLAGS} ${CFLAGS}" \
 	CGO_LDFLAGS="${GODARWINLDFLAGS} ${LFLAGS}" \
+	 MACOSX_DEPLOYMENT_TARGET=${MACOS_MIN_VERSION} \
```

Comment:
Looks like an extra space. Did you use spaces instead of tab? We need single tabs here to please the make.

Reply:
fixed